hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/io/ECKeyOutputStream.java

```diff
@@ -60,7 +59,7 @@
 public class ECKeyOutputStream extends KeyOutputStream {
   private OzoneClientConfig config;
   private ECChunkBuffers ecChunkBufferCache;
-  private int ecChunkSize = 1024;
+  private int ecChunkSize;
   private final int numDataBlks;
   private final int numParityBlks;
   private static final ByteBufferPool BUFFER_POOL = new ElasticByteBufferPool();
```

Comment:
Below there is a hardcodec codec name. You may want to take it from replication config?
private static final String DEFAULT_CODEC_NAME = "rs";